chatmaild/src/chatmaild/tests/test_filtermail.py

```diff
@@ -54,10 +54,14 @@ def test_filtermail_no_encryption_detection(maildata):
 
 
 def test_filtermail_encryption_detection(maildata):
-    msg = maildata("encrypted.eml", from_addr="1@example.org", to_addr="2@example.org")
-    assert check_encrypted(msg)
-
-    # if the subject is not "..." it is not considered ac-encrypted
+    for subject in {"...", "[...]"}:
+        msg = maildata(
+            "encrypted.eml", from_addr="1@example.org", to_addr="2@example.org",
+            replaced_subject=subject
+        )
```

Comment:
for this call to work you need to modify https://github.com/deltachat/chatmail/blob/main/chatmaild/src/chatmaild/tests/plugin.py#L74 as well.  arguably we could use `**kwargs` but that's not the case currently.

Reply:
i'd probably let `maildata()` accept an optional `subject` parameter, and pass it on to the format() call  (formatting is IIRC fine with extra parameters that are not used in a template-string).

Reply:
please do try to run "pytest chatmaild" locally -- this gives more direct feedback compared to CI.

Reply:
the tests didn't pass for me locally (i understood your recommendation to test but didn't have an environment set up that i felt comfortable running the venv you described).  I'm working on sorting that out.

Reply:
truth be told we so far only cared for ./scripts/*.sh invocations -- so far nobody tried to land a PR to chatmaild except the three core maintainers :)
we are also using "black" and "ruff" for formatting.  This is done with "cmdeploy fmt" or "venv/bin/cmdeploy fmt" if you use the environment created by `./scripts/initenv.sh`